spec/models/scheduled_show_spec.rb

```diff
@@ -96,6 +96,10 @@
       Timecop.freeze Time.local(2015)
```

Comment:
this was the source of the intermittent failure - Timecop persists between specs, until you run `Timecop.return`